CMakeLists.txt

```diff
@@ -121,8 +122,11 @@ endif()
 set(TILEDB_LIB_DEPENDENCIES ${TILEDB_LIB_DEPENDENCIES} pthread)
 
 # Add HDFS support
-add_definitions(-DUSE_HDFS)
-set(TILEDB_LIB_DEPENDENCIES ${TILEDB_LIB_DEPENDENCIES} ${CMAKE_DL_LIBS})
+set(HDFS_SOURCE_DIR "${CMAKE_SOURCE_DIR}/deps/HDFSWrapper/hadoop-hdfs-native")
```

Comment:
Is there a way to make this an optional dependency?

Reply:
DONE. USE_HDFS is a cmake option(cached as true).